web/views.py

```
@@ -7018,3 +7037,102 @@ def users_list(request: HttpRequest) -> HttpResponse:
     }
 
     return render(request, "users_list.html", context)
+
+
+@login_required
+@require_POST
+def add_custom_badge(request):
```

Ruff (0.8.2)

`add_custom_badge` is too complex (13 > 10)

(C901)